src/components/Comments.jsx

```diff
@@ -0,0 +1,55 @@
+import { useEffect, useState } from "react";
+import { apiClient } from "../client";
+import { useParams } from "react-router-dom";
+import "../styles/Comments.css";
+import { convertDateLong } from "../utils";
+
+export function Comments({ articleNotFound }) {
+  if (articleNotFound) return <></>;
+  const params = useParams();
+  const article_id = Number(params.article_id);
+
+  const [comments, setComments] = useState([]);
+  const [isCommentsLoading, setIsCommentsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsCommentsLoading(true);
+    apiClient.get(`/articles/${article_id}/comments`).then(({ data }) => {
+      setComments(data.comments);
+      setIsCommentsLoading(false);
+    });
+  }, []);
```

Comment:
Nice, but ideally these requests should be extracted out into functions and required in. to be used without needing to enter the api endpoints like this here. This is because we may be able to re-use the functions in different places.